setup.py

```diff
@@ -23,6 +23,8 @@
 from setuptools import setup
 from setuptools.dist import Distribution
 
+DOCLINES = __doc__.split('\n')
```

Comment:
I hope this will work. I am not sure we have __doc__ methods implemented same as tensorflow/tensorflow